dist/tools/pyterm/pyterm

```diff
@@ -891,6 +931,8 @@ if __name__ == "__main__":
                         action="store_false",
                         help="Do not try to reconnect when failing on "
                              "connection setup (Default)")
+    parser.add_argument("native_tap", nargs=1)
```

Comment:
IIRC, there are and may be applications that use more than one tap interface.

Reply:
e.g. `examples/gnrc_networking_subnets`

Reply:
Thanks for the hint. I will look into that.